Failure on mapIntermediaryJar

When I try to run gradlew yarn it always give a error at :mapIntermediaryJar
```> Task :mapIntermediaryJar
:mapping minecraft to intermediary

> Task :mapIntermediaryJar FAILED

FAILURE: Build failed with an exception.

* Where:
Build file 'D:\ADM\Desktop\Codes\MCP\yarn-1.15.2\build.gradle' line: 628

* What went wrong:
Execution failed for task ':mapIntermediaryJar'.
> Failed to remap jar
```
I already tried to redownload the source folder.

Having the same issue:
```
> Task :mapIntermediaryJar FAILED
:mapping minecraft to intermediary

FAILURE: Build failed with an exception.

* Where:
Build file '/home/ej/Documents/Tools/Yarn1_15/build.gradle' line: 628

* What went wrong:
Execution failed for task ':mapIntermediaryJar'.
> Failed to remap jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.0.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 23s
```

edit: ok, so it gets more interesting - here's the exception https://hasteb.in/yacovipu.kotlin It says the file does not exist - which is bizarre, because it definitely does?

--- a/build.gradle
+++ b/build.gradle
@@ -12,7 +12,7 @@ buildscript {
 		classpath "commons-io:commons-io:2.6"
 		classpath "com.google.guava:guava:28.0-jre"
 		classpath 'de.undercouch:gradle-download-task:3.4.3'
-		classpath 'net.fabricmc:tiny-remapper:+'
+		classpath 'net.fabricmc:tiny-remapper:0.2.1.63'
 	}
 }
 

